services/static-webserver/client/source/class/osparc/data/model/NodeProgressSequence.js

```diff
@@ -20,8 +20,7 @@
  *
  * [CLUSTER_UP_SCALING]
  * [SIDECARS_PULLING]
- * [SERVICE_OUTPUTS_PULLING, SERVICE_STATE_PULLING] (notice the parallelism here)
- * [SERVICE_IMAGES_PULLING]
+ * [SERVICE_OUTPUTS_PULLING, SERVICE_STATE_PULLING, SERVICE_IMAGES_PULLING] (notice the parallelism here)
  * [SERVICE_INPUTS_PULLING] (when this happens, the frontend has already loaded the service and is displaying it to the user) I would still keep it as is, when we decide to make inputs pulling part of the boot sequence this will be helpful.
```

Comment:
hmm. minor. but from our tests we saw this happens twice in a row. I thought this was once from the backend side and once from the frontend. will check it.